src/Sarif.Sarifer/BackgroundAnalysisTextViewCreationListener.cs

```diff
@@ -77,7 +83,8 @@ public void TextViewCreated(ITextView textView)
 
         private void TextBufferViewTracker_FirstViewAdded(object sender, FirstViewAddedEventArgs e)
         {
-            this.backgroundAnalysisService.Value.StartAnalysisAsync(e.Path, e.Text).FileAndForget(FileAndForgetEventName.BackgroundAnalysisFailure);
+            Task task = this.backgroundAnalysisService.Value.StartAnalysisAsync(e.Path, e.Text, e.CancellationToken);
```

Comment:
> e.CancellationToken); [](start = 95, length = 22)

FirstViewAddedEventArgs should contain the "tracking information" and what should be passed in for the CancellationToken would be something like TextBufferViewTrackingInformation.TokenSource.Token.